Carbon.WebApplication.SolutionService/Services/SolutionRegistrationService.cs

```diff
@@ -39,7 +39,7 @@ public async Task AddAsSolution(SolutionCreationRequest data)
                 throw new IncompatibleSolutionException((int)ExceptionEnums.SolutionNameNull, "SolutionCreationRequest SolutionName cannot be Null or Default");
             }
 
-            var sendEp = await _busControl.GetSendEndpoint(new Uri("queue:solution-creation-state"));
```

Comment:
Should we need to check is high avaible queue enabled?

Reply:
Quorum and the classic one works best with exchange: flag. Otherwise, quorum one throws exception.